### ISSUE TITLE
중간 평가 피드백 (이 이슈는 closed 하지 말아요.)

### ISSUE THREAD

**Issue by yuhanappdev** [rhsekdl12/BabsangApp#12]
먼저, 익숙치 않은 툴들을 사용해서 작성하느라 고생많았어요. ​

피드백 하기 전에 강의에서 얘기했던 칸반보드와 기획서의 용도에 대해서 다시 언급하면 칸반보드는 작성하는 시점에서 향후에 해야될 모든 일들을 계획하면서 전체 일정과 범위를 계산하는 것에 큰 의미가 있어요. 전체를 알아야 부분을 언제까지 해야되고 이 계획이 잘 될지 안될지 가늠할 수 있습니다. ​

기획서는 나 자신 그리고 제3자와 소통하기 위한 수단입니다. 즉, 최대한 모든 사항이 적혀 있어야 해당 사항에 대해서 3자 혹은 나 자신이 생각하는 것을 명확하게 이해할 수 있어요. 특히 현업에서는 기획리뷰를 모든 유관부서와 함께 하는데 모든 사항이 적혀있지 않다면 기획 리뷰 시 많은 의문과 대답으로 필요없는 시간을 버릴 수 있어요. ​

칸반보드는 우선 완료된 항목들은 closed 로 변경해서 관리하는 것은 매우 잘했어요. 진행되는 일정 간 진행 상태가 한눈에 보이기 때문에 잘 사용했습니다. 아쉬운 부분은 우선 첫번째는 이슈에 기재하고 특히 프로젝트의 칸반보드와 연계하는 항목은 해야되는 작업들 위주여야 되요.  관리되는 이슈 항목들을 보면 보관(창고) 항목에 있는 것들은 굳이 이슈가 아니라 위키의 별도 페이지를 만들고 거기서 관리하는 것이 훨씬 좋아 보이네요. ​

두번째는 각 항목들의 범위가 너무 커요 예를 들어 (레시피_확보_개발) 이라는 항목은 여러 개의 세부 개발 항목으로 나뉠 수 있을 것 같은데 그 기준은 항 항목당 1일 이상 작업이 소요되면 안됩니다. 즉, 1일 작업량을 기준으로 항목을 나눠서 관리해야 원활히 관리될 수 있어요. ​

기획서 상에서 기본밥상, 다이어트, 업소, 차쥐혼밥러에 따라 메뉴가 어떻게 달라진 다는 것인지 명확하지 않네요. 그리고 레시피들을 모아서 어떻게 앱내에서 관리하고 저장하고 보여줄 것인지에 대한 내용도 추가되면 더 좋겠어요. ​

고생했어요.  이미 중간 평가는 지났지만 점수를 떠나서 현업에서 자주 쓰이는 것들이니 왜 저것을 사용하는지 기획서는 왜 사용하는지 위에 얘기한 의도를 잘 고민해보고 그에 맞는지 더 고민해서 나중에 현업에 갔을 때 많은 도움이 되길 바래요.

**Comment by rhsekdl12** [comment]
감사합니다 교수님!
피드백 주신내용 잘 읽어보았습니다. 
수정할 부분은 수정하고 보완할 부분은 보완하여, 완벽하진 않더라도 최선을 다해서 해보겠습니다
감사합니다!!

**Comment by yuhanappdev** [comment]
예 :) 아직 학생이니 점수도 물론 중요하지만 무엇을 경험하고 해봤는지도 중요해요. 기말까지 완성될 앱은 더 좋아지길 기대할께요.
